webapp-django/crashstats/documentation/jinja2/documentation/supersearch/examples.html

```diff
@@ -14,7 +14,7 @@ <h2 id="list-of-signatures">List of signatures</h2>
 
     <h3>Solution</h3>
     <div class="example">
-        <pre><code><span class="url-domain">{{ full_url(request, 'api:model_wrapper', model_name='SuperSearch') }}</span><span class="query-string">?product={{ product }}&amp;version={{ releases[product][0].version }}&amp;_facets=signature&amp;_facets_size=20</span></code></pre>
+        <pre><code data-url="{{ full_url(request, 'api:model_wrapper', model_name='SuperSearch') }}?product={{ product }}&amp;version={{ releases[product][0].version }}&amp;_facets=signature&amp;_facets_size=20"><span class="url-domain">{{ full_url(request, 'api:model_wrapper', model_name='SuperSearch') }}</span><span class="query-string">?product={{ product }}&amp;version={{ releases[product][0].version }}&amp;_facets=signature&amp;_facets_size=20</span></code></pre>
```

Comment:
Nope, there can be several ``<code>`` tags in the same example, see the "Many results" example below for example.